services/web/project/app/views.py

```diff
@@ -1,16 +1,53 @@
-from flask import Blueprint, render_template
-from flask.helpers import send_from_directory
+from flask import Blueprint, render_template, redirect, url_for
+from flask.helpers import send_from_directory, flash
 from flask_login import login_required, current_user
+from project.app.models import *
+from project.app.forms import NewQuestionForm
 
 views = Blueprint('views', __name__)
 
-
 @views.route('/')
 @login_required
```

Comment:
you can remove auth for home, we should allow `/` and `/questions` for unauthenticated users. `/` was restricted for the development purposes because we did not have any pages before 👍